ospd/server.py

```diff
@@ -46,7 +46,11 @@ def __init__(self, sock: socket.socket, stream_timeout: int):
     def close(self):
         """ Close the stream
         """
-        self.socket.shutdown(socket.SHUT_RDWR)
+        try:
+            self.socket.shutdown(socket.SHUT_RDWR)
+        except OSError as e:
+            logger.debug("Error: Connection closed. %s", e)
```

Comment:
Is this an error while the socket is being closed? Wouldn't self.socket.close() also throw afterwards? At least i would update the log message to mention that this error is ignored. E.g.
```suggestion
            logger.debug("Ignoring error while shutting down the connection. %s", e)
```